cmake/modules/SwiftSource.cmake

```diff
@@ -485,9 +498,8 @@ function(_compile_swift_files
     add_custom_command_target(
         module_dependency_target
         COMMAND
-          "${line_directive_tool}" "${source_files}" --
+          "${PYTHON_EXECUTABLE}" "${line_directive_tool}" "${file_path}" --
           "${swift_compiler_tool}" "-emit-module" "-o" "${module_file}" ${swift_flags}
-          "${source_files}"
```

Comment:
I think that I would rather that we keep the old behavior in tact.  Can you use "@${file_path}" here?